packages/terra-application/src/application-base/ApplicationBase.jsx

```diff
@@ -40,26 +43,6 @@ const propTypes = {
    * If the `locale` prop is not provided, the preferred language from the browser will be used.
    */
   locale: PropTypes.string,
-  /**
-   * Custom translations for the current locale.
-   */
-  customTranslatedMessages: (props, propName, componentName) => {
-    if (!props[propName]) {
-      return null;
-    }
-
-    if (Object.keys(props[propName]).length !== 0 && props.locale === undefined) {
-      return new Error(`Missing locale prop for ${propName} in ${componentName} props`);
-    }
-
-    return null;
-  },
-  /**
-   * The component to render while the translation files are being retrieved.
-   * NOTE: Absolutely no locale-dependent logic should be
-   * utilized in this placeholder.
-   */
-  translationsLoadingPlaceholder: PropTypes.node,
```

Comment:
This is also a breaking change that should be documented in the changelog and upgrade guide. I'm not actually sure why this was getting removed in the terra-application v2 branch. We should verify if we actually want to remove it now as it is not necessary for the react-intl change.